A2 sign up UI

SachiGoto opened the pull request:
# Description
* Sign Up page UI 

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Check List

## URL

## Screenshot
| ![Screenshot 2024-02-13 at 15 51 13](https://github.com/tabi-memo/frontend/assets/70562492/67fc3d82-9e84-4f97-b9ad-89e07f914015) | <img width="498" alt="Screenshot 2024-02-13 at 15 50 51" src="https://github.com/tabi-memo/frontend/assets/70562492/a8767da4-8a66-4c4f-9161-1e6e7b2b8543"> |
| --- | --- |
|Desktop | Mobile |

## Quqesion

In our Figma design, the height of the desktop is 1024px, but typically, a laptop's height is less than 1024px. In the design, the form itself has a height of 832px. Would it be better to adjust the paddings and margins to fit the form on a laptop (within 832px)?

![Screenshot 2024-02-13 at 16 00 13](https://github.com/tabi-memo/frontend/assets/70562492/346313e4-0b5c-4fe4-90b7-6845b739ba49)

SachiGoto committed:
sign up page ui

SachiGoto committed:
restrict to light mode

SachiGoto committed:
Merge branch main into A2_sign_up_UI

SachiGoto committed:
sign up page UI

kanatagu reviewed (commented):
> Quqesion
In our Figma design, the height of the desktop is 1024px, but typically, a laptop's height is less than 1024px. In the design, the form itself has a height of 832px. Would it be better to adjust the paddings and margins to fit the form on a laptop (within 832px)?

Yes, you're right! Definitely we need to adjust it. Sp is fine but could you fix PC overall? 
1. Scroll bar is showing for width but should not be. Should be 100% of width. 
<img width="754" alt="Screenshot 2024-02-17 at 2 04 56 PM" src="https://github.com/tabi-memo/frontend/assets/66394413/1cbc8e25-0d04-4314-b6c0-ed1b27f675a6">

2. When the form box is longer than the normal when the validation errors shows, it seems to be broken. No space top and bottom. 
![screencapture-localhost-3000-signup-email-2024-02-17-14_09_29](https://github.com/tabi-memo/frontend/assets/66394413/db04b4c2-23c3-41bf-ae06-b3b4fda7f74a)

3. And as you said, the form should be displayed without scrolling for the first view. Could you adjust the spacing and height?

SachiGoto committed:
adjusted padding and margin

SachiGoto commented:
> > Quqesion
> > In our Figma design, the height of the desktop is 1024px, but typically, a laptop's height is less than 1024px. In the design, the form itself has a height of 832px. Would it be better to adjust the paddings and margins to fit the form on a laptop (within 832px)?
> 
> Yes, you're right! Definitely we need to adjust it. Sp is fine but could you fix PC overall?
> 
> 1. Scroll bar is showing for width but should not be. Should be 100% of width.
> 
> <img alt="Screenshot 2024-02-17 at 2 04 56 PM" width="754" src="https://private-user-images.githubusercontent.com/66394413/305672425-1cbc8e25-0d04-4314-b6c0-ed1b27f675a6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2NzAxMDMsIm5iZiI6MTcwODY2OTgwMywicGF0aCI6Ii82NjM5NDQxMy8zMDU2NzI0MjUtMWNiYzhlMjUtMGQwNC00MzE0LWI2YzAtZWQxYjI3ZjY3NWE2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIzVDA2MzAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTlmYWNhNmQ0OWVkNDk1ZjBlY2E1MTYyYjNmYmY5NWRkMWJhOGE0NmY0YmRjYzVkZDBhZDUxNmIxY2Y4NTI0ZjcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9iIkrYGnhdWil84f8pA5ViRJlWmSSl3is-kbwqSWwPI">
> 2. When the form box is longer than the normal when the validation errors shows, it seems to be broken. No space top and bottom.
>    ![screencapture-localhost-3000-signup-email-2024-02-17-14_09_29](https://private-user-images.githubusercontent.com/66394413/305672525-db04b4c2-23c3-41bf-ae06-b3b4fda7f74a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg2NzAxMDMsIm5iZiI6MTcwODY2OTgwMywicGF0aCI6Ii82NjM5NDQxMy8zMDU2NzI1MjUtZGIwNGI0YzItMjNjMy00MWJmLWFlMDYtYjNiNGZkYTdmNzRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjIzVDA2MzAwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyY2I2NGM0YzUwYWZjOGRiOTc0MmUyMTUyODE4YmExOWMyOTM2OTUxMzY5ZTZmOGE0NzgyY2Y1YmYwMTM3ZmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.DyuVxwIvvxQYfQDInDjX0kpPixDnPFacq-hZ32rDmKY)
> 3. And as you said, the form should be displayed without scrolling for the first view. Could you adjust the spacing and height?

I'm not sure why a horizontal scroll bar was showing up as I don't see that on my end. I applied 'overflowX = hidden' 
Let me know if you still see the scroll bar. 

I adjusted space around the form and made the form shorter and fixed the issue you pointed out!

kanatagu committed:
Fix signup page

kanatagu reviewed (approved):
I fixed the followings. You can merge to main now!
- The padding was broken for smartphone 
- No need to nest /signup/email anymore so I changed to /signup

SachiGoto committed:
Merge branch 'main' into A2_sign_up_UI

kanatagu committed:
Merge branch 'main' into A2_sign_up_UI
